tests/rptest/services/redpanda.py

```diff
@@ -1829,6 +1828,12 @@ def write_node_conf_file(self,
             pp_keep_alive = self._pandaproxy.cache_keep_alive
             pp_cache_max_size = self._pandaproxy.cache_max_size
 
+        enable_sr = False
+        sr_authn_method = None
+        if self._schema_registry is not None:
+            enable_sr = True
+            sr_authn_method = self._schema_registry.authn_method
```

Comment:
I think it would be clearer to have the template support `SchemaRegistryConfig` directly.